scripts/generate-validations.ts

```diff
@@ -0,0 +1,247 @@
+import fs from 'fs'
+
+import { addresses } from './../src/utils/addresses'
+import { baseAddresses } from './../src/utils/addresses-base'
+
+const LIMIT = 10
+
+const gql = (query: string) => query
```

Comment:
not sure to get what this is for

Reply:
calling the `gql` function enlightens the ide that we're writing graphql